packages/pigeon/lib/cpp_generator.dart

```diff
@@ -901,6 +960,37 @@ EncodableValue ${api.name}::WrapError(const FlutterError& error) {
     indent.newln();
   }
 
+  void _writeClassConstructor(
+      Root root, Indent indent, Class klass, Iterable<NamedType> params) {
+    final Iterable<_HostNamedType> hostParams = params.map((NamedType param) {
+      return _HostNamedType(
+          _makeVariableName(param),
+          getFieldHostDatatype(param, root.classes, root.enums,
+              _shortBaseCppTypeForBuiltinDartType),
+          param.type);
```

Comment:
optional nit: This would look nicer with trailing commas.

Reply:
Done.